JoomGalleryfriends/JG4-dev
# Create assets during migration

This PR is built on top of PR #309, therefore it needs to be merged before merging this one...

This PR solves the issue reported in #306. The Problm was, that no asset entries were created during migration and therefore the permissions check (acl) failed on migrated items.

### How to test this PR
1. You should be able to create, delete, edit content types (categories, images, tags) as before without errors in the backend
2. You should be able to migrate content types without any errors and without PHP warnings afterwards in the list views.
3. Optional: Check that the asset references in the databse are correctly set.
